.github/workflows/republish.yml

```diff
@@ -0,0 +1,16 @@
+name: Publish sequencer.publiclab.org demo
+
+on:
```

Comment:
We should also add manual triggers. We can use the `workflow_dispatch` event. 

Ref https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch